src/CodeGen/Expr/StaticMethodCallExpr.php

```diff
@@ -1,12 +1,10 @@
 <?php
 namespace CodeGen\Expr;
-use Exception;
+
+use CodeGen\ArgumentList;
 use CodeGen\Renderable;
-use CodeGen\Raw;
-use CodeGen\VariableDeflator;
-use LogicException;
 
-class MethodCall implements Renderable
+class StaticMethodCallExpr implements Renderable
 {
```

Comment:
Name is fixed  for file name and  autoloader compatibility. You can rename it for better fit project